### PR TITLE
Cortx 22395 type5 Object operation Negative

### PR DESCRIPTION
**Pull request by akshaym99**
## Problem Statement

-   Type5 Object operation Negative

## Design

-   Design Doc: https://seagate-systems.atlassian.net/wiki/spaces/PRIVATECOR/pages/1116439499/IO+stability+Workload+-+Type-5

## Coding

   Checklist for Author

-   [ ]  Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

## Testing

  Checklist for Author

-   [ ]  New/Affected tests are executed
-   [ ]  Attach test execution logs

## Review 

  Checklist for Author

-   [ ]  JIRA number/GitHub Issue added to PR
-   [ ]  PR is self reviewed
-   [ ]  Jira and state/status is updated and JIRA is updated with PR link
-   [ ]  Check if the description is clear and explained

## Documentation

  Checklist for Author

-   [ ]  Changes done to ReadMe

### CODE REVIEW

**Comment by akshaym99** [comment]
@VikasKumarC @DhobaleVishal Please review/approve

File: scripts/s3/s3api/object_operations_negative.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            except ClientError as err:
+                self.log.info("Get Object exception for non existing object %s", err)
+            try:
+                resp = await self.get_object(bucket=bucket, key=object_key, ranges='bytes=0-9')
+                assert False, resp
+            except ClientError as err:
+                self.log.info("Get Object range read exception for non existing object %s", err)
+            try:
+                resp = await self.delete_object(bucket, object_key)
+                assert False, resp
```
Comment (DhobaleVishal) [inline scripts/s3/s3api/object_operations_negative.py]: Put appropriate error message for assert.

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]
@akshaym99 Check inline comments.

File: scripts/s3/s3api/object_operations_negative.py (hunk 1 above)
Comment (akshaym99) [reply scripts/s3/s3api/object_operations_negative.py]: Added the changes

**Review by DhobaleVishal — APPROVED** [review]

File: workload/s3/s3api/type_5/negative_ops.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ test_1:
 test_2: #negative scenario
   TEST_ID: TEST-NXXXX2
```
Comment (VikasKumarC) [inline workload/s3/s3api/type_5/negative_ops.yaml]: Please update TEST_IDs as per list shared by priyanka

Comment (akshaym99) [reply workload/s3/s3api/type_5/negative_ops.yaml]: Updated

**Review by VikasKumarC — APPROVED** [review]

**Comment by VikasKumarC** [comment]
@akshaym99  Please resolve codacy issues

**Pull request merged**